diesel/src/macros.rs

```diff
@@ -346,3 +346,29 @@ macro_rules! join_through {
         }
     }
 }
+
+#[macro_export]
+macro_rules! debugsql {
+    ($query:expr) => {{
+        use diesel::query_builder::QueryFragment;
+        let mut query_builder = DebugQueryBuilder::new();
+        $query.to_sql(&mut query_builder).unwrap();
+        query_builder.sql
+    }};
+    ($query:ident) => {{
+        use diesel::query_builder::QueryFragment;
+        let mut query_builder = DebugQueryBuilder::new();
+        $query.to_sql(&mut query_builder).unwrap();
+        query_builder.sql
+   }};
+}
+
+#[macro_export]
+macro_rules! printsql {
```

Comment:
I am testing this printsql macro in the server project I set up and if I name both of them debug_sql and print_sql I get:
```
src/models/user.rs:40:5: 40:25 error: argument should be a single identifier
src/models/user.rs:40     print_sql!(command);
                          ^~~~~~~~~~~~~~~~~~~~
error: aborting due to previous error
Could not compile `diesel_test`.
```

Spent a bit of time on this last night and could not figure out why.  Is this some naming convention for macros starting with debug? Or am I missing something simple?